_posts/2020-07-22-grin-node-402-released.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title:  "Grin node patched to v4.0.2"
+author: "@quentinlesceller"
+categories: blog
+---
+
+Grin node v4.0.2 has been released. Binaries are as usual available from the [download section]({{'download' | relative_url}}).
+
+This release includes a patch for a bug in the Node Foreign API. Details in the issue [mimblewimble/grin/#3399](https://github.com/mimblewimble/grin/issues/3399) and the accompanying pull request [mimblewimble/grin/#3400](https://github.com/mimblewimble/grin/pull/3400).
+
+The version 4.x family of releases include many improvements and fixes, and is required in order to interact with the network after the scheduled hard fork, currently estimated to occur on Jul 16 around 11 PM UTC. All users are encouraged to upgrade.
+
+For more details, see the [previous post announcing version 4.0.0]({% post_url 2020-07-02-grin-4-released %}).
```

Comment:
This is no longer "previous". Maybe "For more details see the post announcing version 4.0.0"?